config/manifests/inferenceobjective.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   criticality: Standard
   poolRef:
+    group: inference.networking.k8s.io
```

Comment:
A quick search shows there are many other yamls to update, can you do them all?

<img width="510" height="993" alt="image" src="https://github.com/user-attachments/assets/2a265c7b-0e9b-466d-8f9f-3778f312e4d2" />

Reply:
Updated. I leave conformance test to use v1a2 InferencePool as I think istio and kgateway has not moved to v1 InferencePool yet.

Reply:
pls also update `test/testdata`

Reply:
They already got updated in main https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/test/testdata/inferencepool-with-model-hermetic.yaml#L22, I guess you were looking at the out-dated code repo